pkg/beaconproxy/mongodb.go

```diff
@@ -125,15 +125,12 @@ func (r *repo) Upsert(proxyHostnameMap map[string]*Input) {
 	)
 
 	// loop over map entries (each hostname)
-	for _, entry := range proxyHostnameMap {
-
-		start := time.Now()
-
+	for _, entry := range uconnProxyMap {
 		// pass entry to dissector
 		dissectorWorker.collect(entry)
 
 		// progress bar increment
-		bar.IncrBy(1, time.Since(start))
+		bar.IncrBy(1)
```

Comment:
Yes, I went ahead and updated it in all of the packages. Not sure what sorcery was allowing it to continue working with stuffing in the extra argument. My linter was complaining about it and is what led me to it. This is the function definition that I see in the package:

![image](https://user-images.githubusercontent.com/11990044/129936751-46612c4e-5428-498e-baa2-414fcbd0e91b.png)

Could be something weird with my system too?

Reply:
![image](https://user-images.githubusercontent.com/761220/129970665-21ca0931-c057-416b-acb4-3f62e8cb7331.png)
This is the definition for the function in the mpb package on my machine. It says its held at version 3.3.4. I think you can set up the progress bars to give an estimated finish time, for which it needs the delta times. However, since we don't display anything like that, I think we're good to remove the delta times anyways.